yarn.lock

```diff
@@ -1122,6 +1149,13 @@ brace-expansion@^1.1.7:
     balanced-match "^1.0.0"
     concat-map "0.0.1"
 
+brace-expansion@^2.0.1:
```

Comment:
tsup#sucrase#glob#minimatch" depends on it
   - Hoisted from "tsup#sucrase#glob#minimatch#brace-expansion"